src/storage/src/render/persist_sink.rs

```diff
@@ -225,6 +222,10 @@ where
                             continue;
                         }
 
+                        if should_pause {
```

Comment:
Can we move the rest of the failpoint definition here, close to its usage?

Reply:
no, because I don't want to evaluate an extra environment variable in a hot lop